src/content/en/2022/privacy.md

```diff
@@ -1,22 +1,471 @@
+
+## Established controls to improve visitor's privacy
+
+As websites include a lot of content (scripts, plugins, ...) from third-parties that they might not entirely trust, they might want to protect their users' privacy from these third-parties. Next, we explore the various controls that can be used to restrict the features or data that third-parties have access to, or that make it explicitly clear which information a website wants to obtain from a user.
+
+### Permission Policy
+
+{{ figure_markup(
+  image="permissions-policy-type.png",
+  caption="Permissions Policy usage by API type.",
+  description="Bar chart showing which percentage of websites use permission policy, either the Permissions Policy or Feature Policy. The Feature Policy API was used on 0.69% desktop and 0.52% mobile sites respectively, the Permissions Policy API was found on 2.71% desktop and 2.31% mobile sites and in total 3.26% desktop and 2.73% mobile sites controlled which features their site supported by setting a policy.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRSnqWDZQ07x6NdnIix7JOfg5DqGcoTrrpwaKavCIvNFkIANncyMxahKaTYjLka3rsrcBmrRIURpnPo/pubchart?oid=1921531833&format=interactive",
+  sheets_gid="741173570",
+  sql_file="number_of_websites_with_permissions_policy.sql"
+  )
+}}
+
+By default, any (third-party) script can access the same browser features as the website they're embedded in. In order to limit the features that will be enabled for the website, the website can make use of the <a hreflang="en" href="https://developer.chrome.com/en/docs/privacy-sandbox/permissions-policy/">Permissions Policy</a>. Through an HTTP response header the website can indicate which features it wants enabled. For instance, if the `microphone` feature is not included in this list, none of the scripts embedded in the web page can use it. Although the policy is fairly new, we are seeing an adoption of 2.71% on desktop sites and 2.31% on mobile sites.
+The Permissions Policy supersedes the [Feature Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy), which can still be found on 0.69% of desktop sites and 0.52% of mobile sites.
+By default most of the features regulated by the Permissions Policy are disabled in cross-origin iframes, they can be explicitly enabled through the `allow` attribute. We find that 15.18% of desktop sites and 14.32% of mobile sites make use of this feature. For a more detailed analysis on the use of the `allow` attribute on iframes, please refer to [the security chapter](./security).
+
+{{ figure_markup(
```

Comment:
Image seems to cut off
![image](https://user-images.githubusercontent.com/27865704/192446457-9a2585fe-356a-4927-bafe-fe50ebfe85c0.png)